### PR TITLE
Feat/custom drawer

### PR DESCRIPTION
**Pull request by ashishiiit01**
1. Basic Navigation setup 
2. Implemented Login screen and Onboarding Screen

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,12 +9,22 @@
...
+    "@react-navigation/native": "^5.9.2",
+    "@react-navigation/stack": "^5.14.2",
     "expo": "~40.0.0",
     "expo-status-bar": "~1.0.3",
     "native-base": "^3.0.0-next.19",
     "react": "16.13.1",
     "react-dom": "16.13.1",
     "react-native": "https://github.com/expo/react-native/archive/sdk-40.0.1.tar.gz",
+    "react-native-gesture-handler": "^1.8.0",
+    "react-native-onboarding-swiper": "^1.1.4",
```
Comment (theankurkedia) [inline package.json]: Why did we need this? Can we not implement this manually?

File: src/screen/authScreen/ForgotPassword.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  RootDrawerParamList,
+  'ForgotPassword'
+>;
+
+type Props = {
+  navigation: ProfileScreenNavigationProp;
+};
+export default function ForgotPassword(props: Props) {
+  return (
+    <View flex={1} alignItems={'center'} justifyContent={'center'}>
```
Comment (theankurkedia) [inline src/screen/authScreen/ForgotPassword.tsx]: Can use Center component for this?

File: src/screen/authScreen/ForgotPassword.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import { Text, Button, View } from 'native-base';
+import { StackNavigationProp } from '@react-navigation/stack';
+
+type RootDrawerParamList = {
```
Comment (theankurkedia) [inline src/screen/authScreen/ForgotPassword.tsx]: Why are we defining this in every file?

File: src/screen/authScreen/Login.tsx [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+  Icon,
+  HStack,
+  Input,
+  Button,
+  Image,
+} from 'native-base';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+import { StackNavigationProp } from '@react-navigation/stack';
+
+type RootDrawerParamList = {
```
Comment (theankurkedia) [inline src/screen/authScreen/Login.tsx]: Again?

File: src/screen/authScreen/Login.tsx [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+  'Login'
+>;
+
+type Props = {
+  navigation: ProfileScreenNavigationProp;
+};
+
+export default function Login(props: Props) {
+  return (
+    <View style={{ flex: 1 }}>
```
Comment (theankurkedia) [inline src/screen/authScreen/Login.tsx]: Use Flex component for this

File: src/screen/drawerScreen/channels/index.tsx [hunk 6]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { DrawerNavigationProp } from '@react-navigation/drawer';
+
+import { View, Text, Button } from 'native-base';
+
+type RootDrawerParamList = {
```
Comment (theankurkedia) [inline src/screen/drawerScreen/channels/index.tsx]: Again?

File: package.json (hunk 1 above)
Comment (ashishiiit01) [reply package.json]: I am using only ScrollView for swiper Screen

**Pull request merged**